eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -22,14 +22,62 @@
   <!-- Public API already released with incorrect upper-case naming pattern -->
   <suppress checks="MethodName" files="AsymmetricEncryptionAlgorithm.java"/>
   <suppress checks="MethodName" files="KeyWrapAlgorithm.java"/>
+  <suppress checks="MethodName" files="PartitionSenderImpl.java"/>
   <suppress checks="MethodName" files="SignatureEncoding.java"/>
   <suppress checks="MethodName" files="SymmetricEncryptionAlgorithm.java"/>
   <suppress checks="MethodName" files="RsaSignature.java"/>
 
   <!-- Public API already released with incorrect constant variable naming -->
   <suppress checks="ConstantName" files="AlgorithmResolver.java"/>
+  <suppress checks="ConstantName" files="AmqpErrorCode.java"/>
+  <suppress checks="ConstantName" files="BatchErrorCodeStrings.java"/>
+  <suppress checks="ConstantName" files="ConnectionStringBuilder.java"/>
+  <suppress checks="ConstantName" files="EventProcessorHost.java"/>
+  <suppress checks="ConstantName" files="InMemoryCheckpointManager.java"/>
+  <suppress checks="ConstantName" files="InMemoryLeaseManager.java"/>
+  <suppress checks="ConstantName" files="MessagingFactory.java"/>
+  <suppress checks="ConstantName" files="PartitionManagerOptions.java"/>
+  <suppress checks="ConstantName" files="PartitionScanner.java"/>
   <suppress checks="ConstantName" files="RsaKey.java"/>
   <suppress checks="ConstantName" files="SymmetricKey.java"/>
-  <suppress checks="ConstantName" files="BatchErrorCodeStrings.java"/>
   <suppress checks="ConstantName" files="TaskFailureInformationCodes.java"/>
+
+  <!-- Public API already released with incorrect static variable naming -->
+  <suppress checks="StaticVariableName" files="EventHubClientImpl.java"/>
+
+  <!-- Public API already released with visibility modifier -->
+  <suppress checks="VisibilityModifier" files="BatchOptions.java"/>
+  <suppress checks="VisibilityModifier" files="EventHubClientImpl.java"/>
+
+  <!-- Public API already released with inner assignment-->
+  <suppress checks="InnerAssignment" files="MessageReceiver.java"/>
+
+  <!-- Public API already released with empty block-->
+  <suppress checks="EmptyBlock" files="MessageReceiver.java"/>
+  <suppress checks="EmptyBlock" files="InMemoryLeaseManager.java"/>
+  <suppress checks="EmptyBlock" files="ReactorDispatcher.java"/>
+  <suppress checks="EmptyBlock" files="AzureStorageCheckpointLeaseManager.java"/>
+
+  <!-- Public API already released with redundant modifier-->
+  <suppress checks="RedundantModifier" files="ActiveClientTokenManager.java"/>
```

Comment:
I'm not sure why these ones are suppressed rather than resolved. [ActiveClientTokenManager.java](https://github.com/Azure/azure-sdk-for-java/blob/master/eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/ActiveClientTokenManager.java#L24).  The class is package-private, so its complaining about having a public constructor. I don't think this changes the visibility of this class though because its internal to the package.

@JonathanGiles Thoughts?

Reply:
http://checkstyle.sourceforge.net/config_modifier.html#RedundantModifier

Reply:
I agree. The modifier is redundant and can safely be removed.